Source/Core/Core/API/Events.h

```diff
@@ -44,11 +44,13 @@ struct ClearInterrupt
 };
 struct SaveStateSave
 {
+  SaveStateSave(bool toSlot, int slot) { this->toSlot = toSlot; this->slot = slot; }
   bool toSlot;
   int slot;
 };
 struct SaveStateLoad
 {
+  SaveStateLoad(bool fromSlot, int slot) { this->fromSlot = fromSlot; this->slot = slot; }
   bool fromSlot;
   int slot;
 };
```

Comment:
Please move the constructors out of the header.

The member names should be refactored to `m_toSlot`/`m_fromSlot` and `m_slot`.

Use an initializer list.